jzy3d-jGL/pom.xml

```diff
@@ -13,8 +15,68 @@
 	<properties>
 		<!-- Disable Javadoc Linter -->
 		<additionalparam>-Xdoclint:none</additionalparam>
+		<swt-version>3.116.100</swt-version>
```

Comment:
I think this is not useful anymore as it is defined in the root pom file under name `version.swt`.

Reply:
done